public/app/features/alerting/unified/components/rule-viewer/DeleteModal.tsx

```diff
@@ -69,14 +70,18 @@ export const useDeleteModal = (redirectToListView = false): DeleteModalHook => {
       <ConfirmModal
         isOpen={Boolean(ruleToDelete)}
         title="Delete rule"
-        body="Deleting this rule will permanently remove it from your alert rule list. Are you sure you want to delete this rule?"
+        body={
+          isSoftDeleteEnabled
+            ? 'Are you sure you want to delete this rule? This rule will be recoverable from the Recently deleted page.'
+            : 'Deleting this rule will permanently remove it from your alert rule list. Are you sure you want to delete this rule?'
+        }
```

Comment:
I wonder if we should make this distinction. The `Recently deleted` page is only available for admins, so if you're Editor, you won't be able to recover rules easily

Reply:
But they will be able to recover them by asking the admin. Maybe we can add a note: "Ask the admin to recover deleted rules."?

Reply:
I updated with :
`Are you sure you want to delete this rule? This rule will be recoverable from the Recently deleted page by a user with an admin role.`